src/test/java/io/xstefank/wildfly/bot/PRCommitCheckTest.java

```diff
@@ -0,0 +1,113 @@
+package io.xstefank.wildfly.bot;
+
+import io.quarkiverse.githubapp.testing.GitHubAppTest;
+import io.quarkus.test.junit.QuarkusTest;
+import io.xstefank.wildfly.bot.utils.GitHubJson;
+import io.xstefank.wildfly.bot.utils.Util;
+import org.junit.jupiter.api.Test;
+import org.kohsuke.github.GHEvent;
+import org.kohsuke.github.GHRepository;
+
+import java.io.IOException;
+
+import static io.quarkiverse.githubapp.testing.GitHubAppTesting.given;
+import static io.xstefank.wildfly.bot.model.RuntimeConstants.DEFAULT_COMMIT_MESSAGE;
+import static io.xstefank.wildfly.bot.utils.TestConstants.INVALID_COMMIT_MESSAGE;
+import static io.xstefank.wildfly.bot.utils.TestConstants.PR_JSON;
+import static io.xstefank.wildfly.bot.utils.TestConstants.TEST_REPO;
+
+/**
+ * Tests for the Wildfly -> Format -> Commit checks.
+ */
+@QuarkusTest
+@GitHubAppTest
+public class PRCommitCheckTest {
+
+    private static String wildflyConfigFile;
+    private static GitHubJson gitHubJson;
+    private static String commitMessage;
+
+    @Test
+    void testFailedCommitCheck() throws IOException {
+        wildflyConfigFile = """
+            wildfly:
+              format:
+                commit:
+                  enabled: true
+            """;
+        gitHubJson = GitHubJson.builder(PR_JSON).build();
+        commitMessage = INVALID_COMMIT_MESSAGE;
```

Comment:
these 2 assignments might be moved to `BeforeEach` setup method or just be assigned on declaration as they do not change.